src/api/helpers/get-account-by-field/index.ts

```diff
@@ -27,7 +27,23 @@ const getAccountByField = async (context: Context, field: string, value: string)
 
     const account = accountsArray[0] as Account;
 
-    return account;
+    const accountStatus = await context.db.AccountStatus.findOne({
+      where: { id: account.accountStatusId },
+      take: 1,
+    });
+
+    /**
+     * constructs full account object
+     * adds accountStatus data
+     */
+    const fullAccount = {
+      ...account,
+      accountStatus: {
+        ...accountStatus,
+      },
+    };
```

Comment:
We could do `status` instead of `accountStatus`, so when references in other places its just `account.status` rather than `account.accountStatus`, what do you think? Would also make this simpler

```js
const fullAccount = {
  ...account,
  status: accountStatus,
};
```